modules/lib/src/main/resources/i18n/dialogs.properties

```diff
@@ -102,6 +102,9 @@ dialog.compareVersions.previousVersion=Previous version
 dialog.compareVersions.nextVersion=Next version
 dialog.compareVersions.newestVersion=Newest version
 dialog.compareVersions.versionSubName={0} by {1}
+dialog.publishedChanges.header=Changes since publishing
+dialog.publishedChanges.showUnchanged=Show entire content
```

Comment:
This text is already used in the Compare Versions dialog, so we need to use the same localisation text in both dialogs.